docs/templates/index.md

```diff
@@ -69,14 +69,37 @@ coder templates create <template-name>
 > [examples/](https://github.com/coder/coder/tree/main/examples/templates)
 > directory in the repo.
 
-## Configure Max Workspace Autostop
+## Template policies (enterprise)
 
-To control cost, specify a maximum time to live flag for a template in hours or
-minutes.
+By default, developers have unlimited workspace usage and can use any template version. Template policies allows you to:
 
-```sh
-coder templates create my-template --default-ttl 4h
-```
+- Auto-stop workspaces after 3 hours of inactivity.
```

Comment:
nit: these clauses shouldn't end with periods since they're not complete sentences